### PR TITLE
kie-issues#755: Fix kogito operator nightly deploy pipelines

### PR DESCRIPTION
**Pull request by rodrigonull**
ISSUES: 
- https://github.com/apache/incubator-kie-issues/issues/755
- https://github.com/apache/incubator-kie-issues/issues/807

Initial fixes for kogito operator nightly pipeline

### CODE REVIEW

**Review by jstastny-cz — APPROVED** [review]
LGTM.

File: .ci/jenkins/Jenkinsfile.deploy [hunk 1]
```diff
@@ -55,7 +56,8 @@ pipeline {
                     }
 
                     // Verify we have the connectivity to Openshift
-                    openshift.loginOpenshift()
+                    // TODO: Uncomment this line when we have the Openshift credentials
```
Comment (cimbalek) [inline .ci/jenkins/Jenkinsfile.deploy:59]: What do we need to get them? Or is this in progress in any way?

**Review by cimbalek — APPROVED** [review]

File: .ci/jenkins/Jenkinsfile.deploy (hunk 1 above)
Comment (rodrigonull) [reply .ci/jenkins/Jenkinsfile.deploy:59]: AFAIK the OpenShift Instance is only used to run the tests. I'm not sure who has the credentials but we need to figure that out to add those credentials to Apache Jenkins.
@porcelli Do you know who can help us with that?

Comment (porcelli) [reply .ci/jenkins/Jenkinsfile.deploy:59]: we don't have a OpenShift instance available, so better disable this

**Review by porcelli — APPROVED** [review]
lgtm!

**Review by porcelli — APPROVED** [review]

**Pull request merged**